spec.html

```diff
@@ -449,109 +449,68 @@ <h1>Runtime Semantics: Evaluation</h1>
 </emu-clause>
 
   <emu-clause id="sec-private-name-type-and-objects">
-    <h1>PrivateName Objects</h1>
+    <h1>privateName Objects</h1>
```

Comment:
i think this can probably stay capitalized, since they still could have been the result of a constructor (see also, Descriptor objects and Iteration Result objects, and Symbol)

Reply:
Yeah, that's a good point. Will capitalize.